control-plane/subcommand/connect-init/command_ent_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strconv"
 	"testing"
 
-	connectinject "github.com/hashicorp/consul-k8s/control-plane/connect-inject"
+	connectinject "github.com/hashicorp/consul-k8s/control-plane/connect-inject/controllers/endpoints"
```

Comment:
should we update this `connectInject` to `endpoints`? this is the case in a few of the files.

Reply:
yes good catch!!!